projects/kit/components/slider/slider.style.less

```diff
@@ -214,21 +214,22 @@
 }
 
 :host-context(tui-textfield) :host([type='range']) {
-    --tui-radius: var(--tui-radius-m);
-    @gap: calc(var(--tui-radius) / 2);
+    --t-gap: 1rem;
 
     top: 100%;
-    left: @gap;
-    right: @gap;
-    inline-size: calc(100% - @gap * 2); // Double check Firefox if you want to set `width: auto`
+    left: var(--t-gap);
+    right: var(--t-gap);
+    inline-size: calc(100% - var(--t-gap) * 2); // Double check Firefox if you want to set `width: auto`
```

Comment:
Is this still relevant in Firefox? How about `inline-size: stretch`?

Reply:
> How about inline-size: stretch?

Not works :(

Reply:
I will add `TODO`